rfhub2/cli/rfhub_importer.py

```diff
@@ -87,8 +87,9 @@ def create_collections(self, paths: Set[Path]) -> List[Dict]:
             try:
                 collection = self.create_collection(path)
                 collections.append(collection)
-            except DataError:
+            except (DataError, SystemExit) as ex:
```

Comment:
Ugly but needed. I did a check on libraries at workplace, and a lot of scripts caused issue. It's neither a fault of app nor LibDocPckg, but it's good to be prepared for such scenario.

Reply:
Can you present the scenario showing why it needs to be handled by SystemExit?

Reply:
Also, could you add unit test(s) that would cover this scenario? (DataError and ideally also something like you observed on real life execution)

Reply:
> Can you present the scenario showing why it needs to be handled by SystemExit?

Yes, the situation occured when RfHub was trying to import .py script that was using argparse and was not a file that could be translated into LibDoc.

> Also, could you add unit test(s) that would cover this scenario? (DataError and ideally also something like you observed on real life execution)

I will prepare scripts and unittests for such scenarios.